internal/fptest/ports.go

```diff
@@ -15,11 +15,33 @@
 package fptest
 
 import (
+	"fmt"
 	"sort"
+	"testing"
 
 	"github.com/openconfig/ondatra"
 )
 
+// PortChannelName returns the port channel interface name which is
+// vendor specific.
+func PortChannelName(t testing.TB, dut *ondatra.DUTDevice, i int) string {
+	t.Helper()
+	if i <= 0 {
+		t.Fatalf("port-channel index must be >= 1: %d", i)
+	}
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
+		return fmt.Sprintf("Port-Channel%d", i)
+	case ondatra.CISCO:
+		return fmt.Sprintf("Port-channel%d", i)
```

Comment:
For Cisco IOS XR, the interface would be "Bundle-Ether%d".

Reply:
Thanks! I updated it.